.github/CODEOWNERS

```diff
@@ -5,9 +5,9 @@
 /src/billing/ @DataBiosphere/broad-core-services
 /src/groups/ @DataBiosphere/broad-core-services
 /src/pages/EnvironmentsPage/ @DataBiosphere/manifold-analysis
-/src/constants/datasets.js @DataBiosphere/jadeteam
+/src/constants/datasets.js @DataBiosphere/manifold-data-management
 /src/images/brands/scientificServices/ @DataBiosphere/teaspoons-developers
-/src/import-data/ @DataBiosphere/jadeteam @DataBiosphere/broad-core-services
+/src/import-data/ @DataBiosphere/manifold-data-management @DataBiosphere/manifold-data-management
```

Comment:
nit: only list `@DataBiosphere/manifold-data-management` once

Reply:
Ah, thank you